src/state/types/ethers-contracts/factories/Abi__factory.ts

```diff
@@ -0,0 +1,960 @@
+/* Autogenerated file. Do not edit manually. */
+/* tslint:disable */
+/* eslint-disable */
```

Comment:
Could you please add manual/link to readme how to generate this files?

Reply:
Left instructions in Readme.md